docs/content/contributing/Networking.md

```diff
@@ -0,0 +1,60 @@
+# Networking
+
+We using [Okhttp](https://square.github.io/okhttp/) sometimes with [Retrofit](https://square.github.io/retrofit/) for
+networking in gradle plugins and android.
+
+## Obtain an OkHttpClient
+
+Add dependency on `http-client` module:
+
+```kotlin
+dependencies {
+    implementation(project(":common:http-client"))
+}
+```
+
+Create an instance of `HttpClientProvider`
+
+```kotlin
+val httpClientProvider = HttpClientProvider(statsdSender, timeProvider)
+```
+
+Get client
+
+```kotlin
+val httpClient = httpClientProvider.provide().build()
+```
+
+Method provide() returns `OkHttpClient.Builder` so you can configure it further
+
+## Required tag
```

Comment:
Is it on the same level in menu as "Obtain an OkHttpClient" scenario?
It looks like the third level.